libr/core/disasm.c

```diff
@@ -5075,8 +5068,10 @@ static char *ds_sub_jumps(RDisasmState *ds, char *str) {
 			name = fcn->name;
 		}
 	} else if (f) {
-		RBinReloc *rel;
-		rel = r_core_getreloc (ds->core, ds->analop.addr, ds->analop.size);
+		RBinReloc *rel = NULL;
+		if (!ds->core->bin->is_reloc_patched) {
```

Comment:
@GustavoLCR I saw this was done by you in https://github.com/radareorg/radare2/pull/14587 to correctly show relocations when `-e io.cache=true` is not used. However, when that's used this causes issues because the relocations AFAIK change positions.